js/Grid/Grouping.md

```diff
@@ -90,7 +90,7 @@ $(function () {
 
 The following output is displayed as a result of the above code example.
 
-![](Grouping_images/Grouping_img3.png)
+![Multi column in JavaScript Grid](Grouping_images/javascript-grid-multi-column.png)
```

Comment:
[Multi column Grouping in JavaScript Grid](Grouping_images/javascript-grid-multi-column-grouping.png)

Check and change all places

Reply:
done